code/modules/mob/living/silicon/robot/robot_mob.dm

```diff
@@ -346,10 +346,11 @@ GLOBAL_LIST_INIT(robot_verbs_default, list(
 	var/static/list/special_modules = list(
 		"Combat" = image('icons/mob/robots.dmi', "security-radial"),
 		"Security" = image('icons/mob/robots.dmi', "security-radial"),
-		"Destroyer" = image('icons/mob/robots.dmi', "droidcombat"))
+		"Destroyer" = image('icons/mob/robots.dmi', "droidcombat"),
+		"Hunter" = image('icons/mob/robots.dmi', "xeno-radial"))
 
 	if(mmi?.alien)
-		return list("Hunter" = image('icons/mob/robots.dmi', "xeno-radial"))
+		force_modules = list("Hunter", "Engineering", "Janitor", "Medical", "Mining", "Service") // standard PLUS hunter
```

Comment:
Not sure if this is the way to go. Since effectively, you are overriding a field which isn't really meant to be overridden. The override applies the `standard_modules` list plus one additional module.

It would break borgs which have a `force_modules` defined. Since those will be removed for them, making them unable to pick it.
Prob better to make a local list here instead which contains the modules you can pick from by default initially. Add the hunter to said list and then apply the `force_modules` on said list. Be sure to do a copy of the `standard_modules` instead of a direct assign

Reply:
can an alien borg have a defined `force_modules`?

Reply:
should be improved